docker/0.90-1/base/Dockerfile.cpu

```diff
@@ -11,6 +11,17 @@ RUN cd /tmp && \
      python3 get-pip.py && \
      rm get-pip.py
 
+# Install tensorio
+RUN echo 'installing miniconda'
+RUN curl -LO http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+RUN bash Miniconda3-latest-Linux-x86_64.sh -bfp /miniconda3
+RUN rm Miniconda3-latest-Linux-x86_64.sh
+ENV PATH=/miniconda3/bin:${PATH}
+RUN conda update -y conda
+RUN conda install protobuf tbb libiconv
```

Comment:
You don't need this line anymore. `mlio-py` installs them as its dependencies.

Reply:
Great, thanks for catching that. I'll update the tox.ini as well.